tests/test_vote_2025_04_08_hoodi.py

```diff
@@ -0,0 +1,254 @@
+
+    assert interface.AddNodeOperators(add_node_operators_evm_script_factory).nodeOperatorsRegistry() == simple_dvt
+    assert (
+        interface.AddNodeOperators(add_node_operators_evm_script_factory).trustedCaller()
+        == EASYTRACK_SIMPLE_DVT_TRUSTED_CALLER
+    )
+    assert (
+        interface.ActivateNodeOperators(activate_node_operators_evm_script_factory).nodeOperatorsRegistry()
+        == simple_dvt
+    )
+    assert (
+        interface.ActivateNodeOperators(activate_node_operators_evm_script_factory).trustedCaller()
+        == EASYTRACK_SIMPLE_DVT_TRUSTED_CALLER
+    )
+    assert (
+        interface.DeactivateNodeOperators(deactivate_node_operators_evm_script_factory).nodeOperatorsRegistry()
+        == simple_dvt
+    )
+    assert (
+        interface.DeactivateNodeOperators(deactivate_node_operators_evm_script_factory).trustedCaller()
+        == EASYTRACK_SIMPLE_DVT_TRUSTED_CALLER
+    )
+    assert (
+        interface.SetVettedValidatorsLimits(set_vetted_validators_limits_evm_script_factory).nodeOperatorsRegistry()
+        == simple_dvt
+    )
+    assert (
+        interface.SetVettedValidatorsLimits(set_vetted_validators_limits_evm_script_factory).trustedCaller()
+        == EASYTRACK_SIMPLE_DVT_TRUSTED_CALLER
+    )
+    assert (
+        interface.SetNodeOperatorNames(set_node_operator_names_evm_script_factory).nodeOperatorsRegistry() == simple_dvt
+    )
+    assert (
+        interface.SetNodeOperatorNames(set_node_operator_names_evm_script_factory).trustedCaller()
+        == EASYTRACK_SIMPLE_DVT_TRUSTED_CALLER
+    )
+    assert (
+        interface.SetNodeOperatorRewardAddresses(
+            set_node_operator_reward_addresses_evm_script_factory
+        ).nodeOperatorsRegistry()
+        == simple_dvt
+    )
+    assert (
+        interface.SetNodeOperatorRewardAddresses(set_node_operator_reward_addresses_evm_script_factory).trustedCaller()
+        == EASYTRACK_SIMPLE_DVT_TRUSTED_CALLER
+    )
+    assert (
+        interface.UpdateTargetValidatorLimits(update_target_validator_limits_evm_script_factory).nodeOperatorsRegistry()
+        == simple_dvt
+    )
+    assert (
+        interface.UpdateTargetValidatorLimits(update_target_validator_limits_evm_script_factory).trustedCaller()
+        == EASYTRACK_SIMPLE_DVT_TRUSTED_CALLER
+    )
+    assert (
+        interface.ChangeNodeOperatorManagers(change_node_operator_managers_evm_script_factory).nodeOperatorsRegistry()
+        == simple_dvt
+    )
+    assert (
+        interface.ChangeNodeOperatorManagers(change_node_operator_managers_evm_script_factory).trustedCaller()
+        == EASYTRACK_SIMPLE_DVT_TRUSTED_CALLER
+    )
+
+    # validate vote events
+    assert count_vote_items_by_events(vote_tx, contracts.voting) == 12, "Incorrect voting items count"
+
+    metadata = find_metadata_by_vote_id(vote_id)
+    print("metadata", metadata)
+
+    # assert get_lido_vote_cid_from_str(metadata) == "xxxxx"
+
+    display_voting_events(vote_tx)
```

Comment:
It is not used now, so it's better to remove it to not call `debug_tractTransaction`